pushapkscript/data/pushapk_task_schema.json

```diff
@@ -47,7 +47,12 @@
                     "uniqueItems": true
                 },
                 "google_play_track": {
-                    "type": "string"
+                  "enum": ["production", "beta", "alpha", "rollout"]
+                },
+                "rollout_percentrage": {
```

Comment:
Ouch. I thought the tests would have caught that. I added `"additionalProperties": false`